README.md

```diff
@@ -37,17 +41,17 @@ You can download the dataset from [HuggingFace - CIDAR](https://huggingface.co/d
 
 ```python
 from datasets import load_dataset
-cidar = load_dataset('arbml/CIDAR')
+cidar = load_dataset('arbml/CIDAR-V2')
```

Comment:
We are defaulting to `arbml/CIDAR`.

Reply:
Fixed the issue and updated instruction numbers as Hugging Face.